packages/flutter_tools/lib/src/dart/pub.dart

```diff
@@ -98,9 +98,13 @@ Future<void> pubGet({
     );
     final bool verbose = FlutterCommand.current != null && FlutterCommand.current.globalResults['verbose'];
     final List<String> args = <String>[
-      if (verbose) '--verbose' else '--verbosity=warning',
+      if (verbose)
+        '--verbose'
+      else
+        '--verbosity=warning',
       ...<String>[command, '--no-precompile'],
-      if (offline) '--offline',
+      if (offline)
+        '--offline',
```

Comment:
nit: I find these harder to read this way, particularly since these blocks could potentially be multiple lines.  We should probably get something in the style guide around how we want to do this for the repo...

Reply:
Yeah I dunno what the best style is for these. They're not super readable either way...

I'll revert these. I don't feel strongly about it.